Samples/Xbox/InteractivitySample.cpp

```diff
@@ -32,6 +33,7 @@ namespace
     const int c_joystickXByParticipantReadingLabel          = 1019;
     const int c_joystickYByParticipantReadingLabel          = 1020;
 
+    // TODO: Replace the Project Version ID with your Project Version ID from Interactive Studio

```

Comment:
Actually this "TODO" is on purpose. It's  sample so the message is for the developer downloading the sample.

---
In reply to: [132342571](https://github.com/mixer/interactive-cpp/pull/25#discussion_r132342571) [](ancestors = 132342571)